code/lib/cli/src/generators/baseGenerator.ts

```diff
@@ -358,8 +364,7 @@ export async function baseGenerator(
       );
 
       if (hasEslint && !isStorybookPluginInstalled) {
-        const shouldInstallESLintPlugin = await suggestESLintPlugin();
-        if (shouldInstallESLintPlugin) {
+        if (skipPrompts || (await suggestESLintPlugin())) {
```

Comment:
that is almost what i did. to bypassing the prompts, however we agreed, we still need to fix this issue.